architecture/architecture-rhcos.adoc

```diff
@@ -5,5 +5,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 include::modules/rhcos-about.adoc[leveloffset=+1]
+ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
This doesn't seem to be having an effect in the preview

Reply:
It does - it shows the "About" RHCOS/Ignition, but does not go into configuration/troubleshooting. Did we want to just axe the whole page?

Reply:
Yeah, probably

Reply:
Updated the commit to exclude this page, thanks